fontreport/fontreport.py

```diff
@@ -606,14 +606,14 @@ def main():
         tofile = name + '.tex'
 
     with open(tofile, 'w') as f:
-      f.write(envelope.Report(xetex).encode('utf-8'))
+      f.write(envelope.Report(xetex))
```

Comment:
Python does implicit encoding of unicode string into one of supported encoding depending on runtime environment (current locale, is output a tty or not), while xelatex expects utf-8 encoding as an input. While existing reports work fine with this change, I suggest to keep an 'utf-8' encoding here (or pass encoding='utf-8' parameter to open) for consistency. Otherwise any data in a report that do not fit into latin1 range (potentially - unicode entries from 'name' table) can break .pdf report generation.